docs/docs/cmd/spo/homesite/homesite-list.mdx

```diff
@@ -0,0 +1,107 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# spo homesite list
+
+Lists all the Home Sites
+
+## Usage
+
+```sh
+m365 spo homesite list [options]
+```
+
+## Options
+
+<Global />
+
+## Examples
+
+Lists all the Home Sites
+
+```sh
+m365 spo homesite list
+```
+
+## Response
```

Comment:
When listing responses, to avoid cluttering, let's just display one item (instead of 2 now) for every response type.